src/CRRao.jl

```diff
@@ -37,16 +76,228 @@ PoissonRegression
 ```
 
 Type representing the Poisson Regression model class.
+```math
+y_i \\sim Poisson(\\lambda_i), i=1,2,\\cdots,n
+```
+where
+
+```math
+\\lambda_i = \\exp(\\alpha +\\mathbf{x}_i^T\\beta),
+```
++ ``y_i`` is the ``i^{th}`` element of the response vector ``y``,
++ ``\\mathbf{x}=(x_{i1},x_{i2},\\cdots,x_{in})`` is the ``i^{th}`` row of the design matix of size ``n \\times p``,
++ ``\\alpha`` is the intercept of the model, and
++ ``\\beta`` is the regression coefficients of the model.
+
 """
 struct PoissonRegression end
 
 
 struct Prior_Gauss end
```

Comment:
@sourish-cmi Prior_Gauss was added to CRRao and needs documentation.

Reply:
Yes - I just committed the documentation for the `Prior_Gauss`. Please have a look.